dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderTelemetry.h

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <WindowsAppRuntimeInsights.h>
+#include <NotificationTelemetryHelper.h>
+
+DECLARE_TRACELOGGING_CLASS(AppNotificationBuilderTelemetryProvider,
+    "Microsoft.WindowsAppSDK.Notifications.AppNotificationBuilderTelemetry",
+    // {6f23f3a8-1420-4814-83c7-c752565aad22}
+    (0x6f23f3a8, 0x1420, 0x4814, 0x83, 0xc7, 0xc7, 0x52, 0x56, 0x5a, 0xad, 0x22));
+
+class AppNotificationBuilderTelemetry : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TELEMETRY_CLASS(AppNotificationBuilderTelemetry, AppNotificationBuilderTelemetryProvider);
+
+public:
+    DEFINE_EVENT_METHOD(LogBuildNotification)(
+        winrt::hresult hr) noexcept try
```

Comment:
const &

Reply:
Pass by value should be fine here since winrt::hresult is a POD that takes only 4 bytes. That's half the size of a pointer on a x64 machine.